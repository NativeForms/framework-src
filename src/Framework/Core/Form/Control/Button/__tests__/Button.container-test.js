import React from 'react';
import { shallow } from 'enzyme';
import ButtonContainer from '../Button.container';

describe('container <ButtonContainer />', () => {
  it('container render correctly', () => {
    const wrapper = shallow(
      <ButtonContainer />
    );
    expect(wrapper.dive()).toMatchSnapshot();
  });
});
