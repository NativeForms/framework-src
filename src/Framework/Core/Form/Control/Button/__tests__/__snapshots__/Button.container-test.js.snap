// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`container <ButtonContainer /> container render correctly 1`] = `
<Content
  style={
    Object {
      "backgroundColor": "transparent",
      "flex": 1,
    }
  }
>
  <GridNB>
    <RowNB
      style={
        Object {
          "padding": 10,
        }
      }
    >
      <FormComponent
        messages={
          Object {
            "myLabel": "Button Text",
            "title": "Button",
          }
        }
        schema={
          Object {
            "attributes": Object {
              "formComponents": Array [
                Object {
                  "attributes": Object {
                    "label": Object {
                      "code": "myLabel",
                    },
                  },
                  "category": "control",
                  "type": "button",
                  "uid": "Button1",
                },
              ],
              "label": "Button Example",
            },
            "category": "group",
            "type": "section",
            "uid": "buttonExample",
          }
        }
      />
    </RowNB>
    <RowNB>
      <SchemaViewer>
        export default {
          uid: 'buttonExample',
          type: 'section',
          category: 'group',
          attributes: {
            label: 'Button Example',
            formComponents: [{
              uid: 'Button1',
              type: 'button',
              category: 'control',
              attributes: {
                label: {
                  code: 'myLabel'
                },
              }
            }]
          }
        };
        
      </SchemaViewer>
    </RowNB>
  </GridNB>
</Content>
`;
